runtime/include/chpl-ISO_Fortran_binding.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifdef __ICC
+#include "ISO_Fortran_binding.h"
+#else
+#ifdef _CRAYC
+#include "ISO_Fortran_binding.h"
+#else
+// Stub it out for other compilers
```

Comment:
This allows programs that don't use it to compile without the explicit definitions?

Reply:
As long as they don't try to use anything in ISO_Fortran_binding.chpl it gets removed and nothing is needed from this header.  The full test suite passed with this blank header.